itests/src/test/resources/testconfiguration.properties

```diff
@@ -7,6 +7,7 @@ minimr.query.files=\
 # Queries ran by both MiniLlapLocal and MiniTez
 minitez.query.files.shared=\
   compressed_skip_header_footer_aggr.q,\
+  create_table.q,\
```

Comment:
1. similarly to HIVE-25331, please use TestMiniLlapLocalCliDriver
2. this patch creates the same .q file, please use a different one, otherwise they'll conflict I guess